code/modules/client/preference_setup/loadout/lists/suits.dm

```diff
@@ -27,10 +29,11 @@
 	display_name = "poncho, science"
 	path = /obj/item/clothing/suit/poncho/roles/science
 
-/datum/gear/suit/nanotrasen_poncho
+//NT descriptions
+/*/datum/gear/suit/nanotrasen_poncho
 	display_name = "poncho, NanoTrasen"
 	path = /obj/item/clothing/suit/poncho/roles/science/nanotrasen
-
+*/
 /datum/gear/suit/cargo_poncho
 	display_name = "poncho, supply"
 	path = /obj/item/clothing/suit/poncho/roles/cargo
```

Comment:
Can't comment below but leather coat / trenchcoat would be valid for command security roles, imo, where uniform restrictions would be a mite more relaxed for higher ranking personnel, and also disgingush them, can change it or I may just go add it to their uniform or just enable it for certain roles.

Reply:
Good point, i'll unrestrict it for GC and zonecomms...

Reply:
![image](https://user-images.githubusercontent.com/67012187/179532373-a1c329ac-0403-4fcf-ba97-a7b89d65152f.png)